scripts/contractsgen.sh

```
@@ -1,5 +1,8 @@
+set -e
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)